src/interfaces.ts

```diff
@@ -99,7 +99,7 @@ export interface ChatRequest {
   model: string
   messages?: Message[]
   stream?: boolean
-  format?: string
+  format?: string | object
```

Comment:
Should the GenerateRequest `format` be updated too?

Reply:
yes ty